projects/storefrontlib/src/lib/ui/pages/pages.module.ts

```diff
@@ -86,6 +86,18 @@ const pageModules = [
         path: '**',
         canActivate: [CmsPageGuards],
         component: PageLayoutComponent
+      },
+      {
+        path: null,
+        canActivate: [CmsPageGuards],
+        component: PageLayoutComponent,
+        data: { cxPath: 'product' }
+      },
+      {
+        path:
```

Comment:
@Platonn this might be of interest of you. Would you mind talking to @Pucek9 and align with the changes?